lib/ssh_scan/database/mongo_queue.rb

```diff
@@ -0,0 +1,55 @@
+require 'mongo'
+
+Mongo::Logger.logger.level = ::Logger::FATAL
+
+module SSHScan
+  module QueueDB
+    class MongoDb
+      attr_reader :queue_collection
```

Comment:
My concern with the existing approach is that it created a lot of new code and maybe that's not needed for this additional function.  Really all we're doing is adding an additional table for queue info.